CITATION.cff

```diff
@@ -85,7 +85,7 @@ authors:
     orcid: https://orcid.org/0000-0002-0503-9348
 title: "Reproducible image-based profiling with Pycytominer"
 # This version is updated using `cz bump` command
-version: "1.2.1"
+version: "1.2.2"
```

Comment:
Should this be 1.3 b/c of bug fix? Ultimately, it is your call but I would advocate for this (i can be easily convinced otherwise)

Reply:
Thanks, good question. I followed [semver.org](https://semver.org/) semantic version numbering here and would suggest we stick with it (I feel it's standard practice in many places and is relatively uncomplicated).

```
MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes
```

We should probably formalize this somewhere in the docs to stipulate that we follow this or another scheme (alongside PEP 440 conventions, which are outlined already in the `pyproject.toml` file). I'll create an issue for this.

Reply:
Raised #499 to document this moving forward. For now I'm going to stick with `1.2.2` to try and follow the semver convention.